docs/recipes/build/add-build-script.md

```diff
@@ -1,88 +1,256 @@
-# How do I add a build script to the project?
+# How do I add build automation to the project?
 
 ## FAKE
 [Fake](https://fake.build/) is a DSL for build tasks that is modular, extensible and easy to start with. Fake allows you to easily build, bundle, deploy your app and more by executing a single command.
 
-> The standard template comes [with a FAKE script](../../../template-fake) by default, and **this recipe only applies to the minimal template**.
+> The standard template comes [with a FAKE project](../../../template-safe-commands) by default, so **this recipe only applies to the minimal template**.
 
-## Paket
-Before moving on, we recommend migrating to [Paket](https://fsprojects.github.io/Paket/). In fact, **this is a prerequisite for this recipe**. It is possible to use FAKE without Paket by creating an executable instead of a script file, however this will not be covered in this recipe.
+---
+#### 1. Create a build project
 
-If you’re *not* using Paket to handle your dependencies, go through the [Migrate to Paket](../../package-management/migrate-to-paket) recipe before continuing.
+Create a new console app called 'Build' at the root of your solution
 
----
-#### 1. The Build Dependencies
-Paste the following at the end of your `paket.dependencies` file, which is at the root of the solution.
-```
-group Build
-    source https://api.nuget.org/v3/index.json
-    framework: netstandard2.0
-    storage: none
-
-    nuget FSharp.Core
-    nuget Fake.Core.Target
-    nuget Fake.DotNet.Cli
-    nuget Fake.IO.FileSystem
+```fsharp
+dotnet new console -lang f# -n Build
 ```
-Then, execute `paket install` to install these dependencies.
 
-#### 2. Fake CLI Tool
-Run the following commands at the root folder of your solution to install FAKE as a local tool:
-```bash
-dotnet tool install fake-cli
-dotnet tool restore
+#### 2. Create a build script
+
+Open the generated folder.
+
+Open the project in your IDE and rename the file `Program.fs` to `Build.fs`.
```

Comment:
Do you need to rename the file - why not just leave it as Program.fs?

Reply:
@isaacabraham Small differences like this between the **default** template and **minimal + recipes** can complicate things down the line.
If you need to reference this file in some docs, you'd have to add a qualifier like "`Build.fs` (or `Program.fs` in the `Build` folder if you're using the minimal template with the add-build-script recipe)..."

The [testing the server](https://github.com/SAFE-Stack/docs/blob/82a25cede08fcad75175c2726772004e44b75531/docs/recipes/build/docker-image.md#4-testing-the-server) part of the docker-image recipe is an example of where this kind of thing has happened.

Not a big deal but it would be nice to minimise unnecessary differences.

Reply:
Perhaps we then just to explain why we're renaming it, otherwise there's a good chance that people will otherwise think that there's some explicit requirement to call it Build.fs.

Reply:
Sorry wasn't getting notifications for this thread.

I agree with Oliver here, and I did it to keep everything aligned.

I also don't think that we need to spell out things like that, if people are savvy they will know and if not it isn't the best place to explain it really.

Reply:
This is also why I was keen to keep the setup the same as the full template here, not cut it right back and take out all the deployment stuff. I thought the idea was to take people from min to max, or max to min. That is the approach all the recipes have taken so far I think.

Reply:
In general I'd agree that we don't need to spell out things like that, except that in this case there IS a requirement with FAKE to name the script `build.fsx`. Maybe a little note like "To make it the same as the standard template, open the project..."

Reply:
After the server and client test recipes are merged I'm thinking of making a PR to change the tests locations to be in line with the standard template.

Reply:
There's no build.fsx any more. We are running a project called Build.fsproj, the module inside could technically be called anything.

Reply:
> After the server and client test recipes are merged I'm thinking of making a PR to change the tests locations to be in line with the standard template.

Good point, I did just chuck them in the `src` folder, I should have kept them consistent as well really and put them in a `tests` folder.

Reply:
I know but I think @isaacabraham is thinking of people who have used FAKE before and are used to it needing a `build.fsx`

Reply:
> This is also why I was keen to keep the setup the same as the full template here, not cut it right back and take out all the deployment stuff. I thought the idea was to take people from min to max, or max to min. That is the approach all the recipes have taken so far I think.

I guess it doesn't hurt to have an 'Add build automation' and an 'Add deployment automation' recipe seperately come to think of it. Shame to cut out all the nice build stuff that was put into the full template now though, with the coloured labelled output etc.

Reply:
Added a note to explain it :)